<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PuttingForm</class>
 <widget class="QWidget" name="PuttingForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Putting Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Putting System</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="putting_system_combo">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>ExPutt Settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Auto start Camera App</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="exputt_camera_auto_start_combo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>New Item</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Camera App Window Title                    </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="exputt_camera_window_title_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Window Title</string>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="tabChangesFocus">
                 <bool>true</bool>
                </property>
                <property name="lineWrapMode">
                 <enum>QPlainTextEdit::WidgetWidth</enum>
                </property>
                <property name="placeholderText">
                 <string>WindowTitle</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="rois_button">
              <property name="text">
               <string>ROI's</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Webcam Settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Camera ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="webcam_camera_combo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>New Item</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Ball Color</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="webcam_ball_color_combo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>New Item</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="webcam_window_width_label">
                <property name="text">
                 <string>Camera Window Width                         </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="webcam_putting_width_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Window Title</string>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="tabChangesFocus">
                 <bool>true</bool>
                </property>
                <property name="lineWrapMode">
                 <enum>QPlainTextEdit::WidgetWidth</enum>
                </property>
                <property name="placeholderText">
                 <string>Camera Window Width</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Auto Start Ball Tracking App</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="webcam_auto_start_combo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>New Item</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="ball_tracking_app_params_label">
                <property name="text">
                 <string>Ball Tracking App Params                      </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="ball_tracking_app_params_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Window Title</string>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="tabChangesFocus">
                 <bool>true</bool>
                </property>
                <property name="lineWrapMode">
                 <enum>QPlainTextEdit::WidgetWidth</enum>
                </property>
                <property name="placeholderText">
                 <string>Ball Tracking App Params</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Ball Tracking App Window Title            </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="webcam_window_title_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Window Title</string>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="tabChangesFocus">
                 <bool>true</bool>
                </property>
                <property name="lineWrapMode">
                 <enum>QPlainTextEdit::WidgetWidth</enum>
                </property>
                <property name="placeholderText">
                 <string>WindowTitle</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Window Behaviour - Putting</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Set Focus To</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="webcam_putting_focus_combo">
                   <item>
                    <property name="text">
                     <string>None</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>New Item</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Window Behaviour - Not Putting</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_14">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Putting Window</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="webcam_not_putting_state_combo">
                   <item>
                    <property name="text">
                     <string>None</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>New Item</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="find_video_sources_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Save Changes</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Find Video Sources</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Save Changes</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Close Form</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
